source/_includes/puppet-collections/_rsk_import_key.md

```diff
@@ -0,0 +1,6 @@
+    gpg: requesting key 4BD6EC30 from hkp server keys.gnupg.net
```

Comment:
This will go out with puppet-agent 1.7.0 and will apply to everything (all PE and FOSS releases) except for PE 3.x. PE 3.x will remain on the old key.